### PR TITLE
Sphere display with various sizes

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
With the option --sphereRadiusFromInput we can customize the size of each sdp point if displayed with sphere primitive (4th column).

# Checklist

- [n.a ] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [n.a] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: visualisation/3dSDPViewer.cpp [hunk 1]
```diff
@@ -74,8 +75,9 @@ int main( int argc, char** argv )
     ("scaleX,x",  po::value<float>()->default_value(1.0), "set the scale value in the X direction (default 1.0)" )
     ("scaleY,y",  po::value<float>()->default_value(1.0), "set the scale value in the Y direction (default 1.0)" )
     ("scaleZ,z",  po::value<float>()->default_value(1.0), "set the scale value in the Z direction (default 1.0)")
-    ("sphereRadius,s",  po::value<double>()->default_value(0.2), "defines the sphere radius (used when the primitive is set to the sphere). (default value 0.2)")
     ("sphereResolution",  po::value<unsigned int>()->default_value(30), "defines the sphere resolution (used when the primitive is set to the sphere). (default resolution: 30)")
+    ("sphereRadius,s",  po::value<double>()->default_value(0.2), "defines the sphere radius (used when the primitive is set to the sphere). (default value 0.2)")
+    ("sphereRadiusFromInput", "takes as sphere radius the 4th field of the sdp input file.")
```
Comment (dcoeurjo) [inline visualisation/3dSDPViewer.cpp]: takes, as sphere radius, ...

File: visualisation/3dSDPViewer.cpp [hunk 2]
```diff
@@ -214,6 +222,10 @@ int main( int argc, char** argv )
                              (int)vectVoxels.at(i)[1],
                              (int)vectVoxels.at(i)[2]);
       }else{
+        if(useMultiRad){
+          sphereRadius = vectSphereRadius[i];
```
Comment (dcoeurjo) [inline visualisation/3dSDPViewer.cpp]: the coding style does not match with the one in CONTRIBUTING ;)

**Comment by kerautret** [comment]
@dcoeurjo corrected, and added new commits to improve color import (explicit color instead index)

**Comment by kerautret** [comment]
@dcoeurjo can I merge ?

**Comment by dcoeurjo** [comment]
Yup ;) thanks

**Pull request merged**